webpages/settings/index.html

```diff
@@ -120,17 +120,24 @@
   </head>
   <body v-cloak :class="{light: theme}">
     <div class="navbar">
```

Comment:
```suggestion
    <div class="navbar" v-if="!isIframe">
```
This will fix autofocus. The code in `style.css` that hides the navbar in the popup (lines 204-206) can then be removed.